### PR TITLE
translated controlled demo

### PR DESCRIPTION
**Pull request by xueqingxiao**

### CODE REVIEW

**Review by afc163 — APPROVED** [review]

File: components/mention/demo/controlled.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ let App = React.createClass({
       <Form horizontal>
         <FormItem
           id="control-mention"
-          label="最帅的码农"
+          label="The most handsome programmer"
```
Comment (afc163) [inline components/mention/demo/controlled.md]: label 不能这么长，会溢出。

File: components/mention/demo/controlled.md [hunk 2]
```diff
@@ -71,9 +71,9 @@ let App = React.createClass({
           )}
         </FormItem>
         <FormItem wrapperCol={{ span: 14, offset: 6 }}>
-          <Button type="primary" onClick={this.handleSubmit}>确定</Button>
+          <Button type="primary" onClick={this.handleSubmit}>Submit</Button>
           &nbsp;&nbsp;&nbsp;
-          <Button type="ghost" onClick={this.handleReset}>重置</Button>
+          <Button type="ghost" onClick={this.handleReset}>Rest</Button>
```
Comment (afc163) [inline components/mention/demo/controlled.md]: Reset

**Review by afc163 — CHANGES_REQUESTED** [review]

File: components/mention/demo/controlled.md (hunk 1 above)
Comment (kvetoslavnovak) [inline components/mention/demo/controlled.md]: Error callback: More than one must be selected!
Label:  Top programmers

File: components/mention/demo/controlled.md (hunk 2 above)
Comment (kvetoslavnovak) [inline components/mention/demo/controlled.md]: Submit
Reset

**Pull request merged**